projects/lib/src/providers/dummy-login-provider.ts

```diff
@@ -90,7 +88,7 @@ export class DummyLoginProvider extends BaseLoginProvider {
   signOut(revoke?: boolean): Promise<any> {
     return new Promise((resolve, reject) => {
       this.loggedIn = false;
-      resolve();
+      resolve(this.dummy);
```

Comment:
Could you please revert this code? I'll fix it in a different PR where I change the method signature.

Reply:
ok